### PR TITLE
Update dockerfile to fit with prow CI

### PR DESCRIPTION
**Pull request by liangxia**
PTAL
/cc @jhou1 @JianLi-RH @dis016 @pruan-rht

### CODE REVIEW

**Comment by liangxia** [comment]
The failure is as below currently, as it rely on https://github.com/openshift/cucushift/pull/8674
```
+ curl -sSL https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm -o /tmp/epel-release-latest-8.noarch.rpm
+ yum install -y /tmp/epel-release-latest-8.noarch.rpm
Loaded plugins: fastestmirror, ovl
Examining /tmp/epel-release-latest-8.noarch.rpm: epel-release-8-11.el8.noarch
Marking /tmp/epel-release-latest-8.noarch.rpm to be installed
Resolving Dependencies
--> Running transaction check
---> Package epel-release.noarch 0:8-11.el8 will be installed
--> Processing Dependency: redhat-release >= 8 for package: epel-release-8-11.el8.noarch
Loading mirror speeds from cached hostfile
 * base: d36uatko69830t.cloudfront.net
 * extras: d36uatko69830t.cloudfront.net
 * updates: d36uatko69830t.cloudfront.net
--> Finished Dependency Resolution
Error: Package: epel-release-8-11.el8.noarch (/epel-release-latest-8.noarch)
           Requires: redhat-release >= 8
           Installed: centos-release-7-9.2009.1.el7.centos.x86_64 (@updates)
               redhat-release = 7.9-3
           Available: centos-release-7-9.2009.0.el7.centos.x86_64 (base)
               redhat-release = 7.9-3
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
error: build error: error building at STEP "RUN set -x &&     yum -y update &&     INSTALL_PKGS="bsdtar git openssh-clients httpd-tools" &&     yum install -y $INSTALL_PKGS &&     GECKODRIVER_DOWNLOAD_URL="$(curl -sSL https://api.github.com/repos/mozilla/geckodriver/releases/latest | grep -Eo 'http.*linux64.tar.gz' | sed -E 's/.*(https[^"]*).*/\1/' | head -1)" &&     curl -sSL $GITHUB_API_CURL_OPTS "$GECKODRIVER_DOWNLOAD_URL" | bsdtar -xvf - -C /usr/local/bin &&     chmod +x /usr/local/bin/geckodriver &&     curl -sSL https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm -o /tmp/epel-release-latest-8.noarch.rpm &&     yum install -y /tmp/epel-release-latest-8.noarch.rpm": error while running runtime: exit status 1
```

**Comment by jhou1** [comment]
Thanks, looks good. Once the cucushift PR is merged and the test passes, we can get this merged

**Comment by liangxia** [comment]
We need to wait for the cached base image refresh

File: tools/openshift-ci/Dockerfile [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+ADD . /verification-tests/
+RUN mv /tierN/ /verification-tests/tierN/
+RUN set -x && \
+    yum module reset ruby && \
+    yum module -y enable ruby:2.7 && \
+    yum module -y install ruby:2.7
+RUN /verification-tests/tools/install_os_deps.sh
+RUN /verification-tests/tools/hack_bundle.rb
 
 RUN yum clean all -y && rm -rf /var/cache/yum /tmp/*
```
Comment (jhou1) [inline tools/openshift-ci/Dockerfile:27]: Maybe remove the Gemfile.lock here?

Comment (liangxia) [reply tools/openshift-ci/Dockerfile]: Made the change and verified it works.

**Comment by liangxia** [comment]
The tests pass. PR ready for review.

**Pull request merged**